DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionTunnelController.swift

```diff
@@ -295,8 +296,7 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
     }
 
     private func handleSetExcludeLocalNetworks(_ excludeLocalNetworks: Bool) async throws {
-        guard let tunnelManager = await manager,
-              tunnelManager.protocolConfiguration?.excludeLocalNetworks == !excludeLocalNetworks else {
+        guard let tunnelManager = await manager else {
```

Comment:
We're not really using `protocolConfiguration.excludeLocalNetworks` anymore.  It'll always be `false`.